src/main/kotlin/ar/edu/unahur/obj2/vendedores/CentroDistribucion.kt

```diff
@@ -0,0 +1,26 @@
+package ar.edu.unahur.obj2.vendedores
+
+class CentroDistribucion(val ciudad: Ciudad) {
+    val vendedores = mutableListOf<Vendedor>()
+    fun agregarVendedor(cual: Vendedor) {
+        if (vendedores.contains(cual)) {
+            throw Exception("ERROR: Vendedor ya registrado")
+        } else {
+            vendedores.add(cual)
+        }
```

Comment:
Si se ejecuta la rama verdadera del `if`, el método explota y nunca llega al `else`. Podés entonces obviarlo, te quedaría así:

```suggestion
        if (vendedores.contains(cual)) {
            throw Exception("ERROR: Vendedor ya registrado")
        }
        vendedores.add(cual)
```